apps/dashboard/app/views/files/_navigate_breadcrumb.html.erb

```diff
@@ -0,0 +1,18 @@
+<% if file_counter == 0 %>
+  <li class="breadcrumb-item">
+    <span id="<%= files_path(@filesystem, Dir.home) %>" class="fa fa-home" style="cursor: pointer;"></span>
+  </li>
+  <li class="breadcrumb-item">
+      <span id="<%= files_path(@filesystem, full_path.parent) %>" class="fa fa-arrow-up" style="cursor: pointer;"></span>
```

Comment:
Let's avoid using inline styles like this `style="cursor: pointer;"`. I'm sure there's a bootstrap class for the same, and if not we can create one.

Reply:
Note we use inline styles a couple times here.

Reply:
While I agree 100% that we shouldn't use inline.  I tried the Bootstrap equivalent.  It does not work in the modal window.  The bootstrap equivalent is `cursor-pointer`

Reply:
When we enable a content security policy, stuff like this will get blocked.